test/load/test_load_tx_ethbridge_burn.py

```diff
@@ -71,25 +145,57 @@ def _test_load_tx_ethbridge_burn(ctx: test_utils.EnvCtx, amount_per_tx: int, tra
     sif_balances_initial: List[cosmos.Balance] = [ctx.get_sifchain_balance(sif_acct) for sif_acct in sif_accts]
     eth_balances_initial: List[str] = [ctx.eth.get_eth_balance(eth_acct) for eth_acct in eth_accts]
     assert all([b == 0 for b in eth_balances_initial])  # Might be non-zero if we're recycling accounts
+    
+    if token_denom != ctx.ceth_symbol:
+        erc20_balances_initial: List[str] = [ctx.get_erc20_token_balance(token_sc.address, eth_acct) for eth_acct in eth_accts]
+        assert all([b == 0 for b in eth_balances_initial])  # Might be non-zero if we're recycling accounts
+
 
     # Create a dispensation sif account that will receive all locked ETH and dispense it to each sif account
     # (we do this in one transaction because lock transactions take a lot of time).
     # Dispensation account needs rowan for distributing ceth to sif_accts.
-    dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: n_sif * sif_tx_fee_in_rowan})
+    if token_denom == rowan:
+        amount = sum_all * amount_per_tx + n_sif * 2 * sif_tx_fee_in_rowan
+        dispensation_sif_acct: cosmos.Address = ctx.create_sifchain_addr(fund_amounts={rowan: amount})
+    elif token_denom == ctx.ceth_symbol:
```

Comment:
That ceth symbol there, what token does it represent?

Is it cETH from Compound, does it represent the native token of whatever chain we are interacting with, or something else?

Reply:
The denom is from peggy1.0. We just add 'c' as prefix for any token imported from Ethereum. Ceth represents the eth in sifnode.